wpeverest/everest-forms
# Setting Enhancement - Image field added

### All Submissions:






### Changes proposed in this Pull Request:



Image Field is added for settings.

### How to test the changes in this Pull Request:

1. GOTO Setting(Global) 
2. Select PDF
3. You can see select header logo option there.

### Types of changes:




### Other information:




### Changelog entry

> Setting Enhancement - Image field added
